tests/core/test_processor.py

```diff
@@ -260,18 +275,20 @@ def test_processor_get_legend_entries():
     assert len(after_process) == 1
 
 
-def test_processor_sort_by_law_status():
+def test_processor_sort_by_law_status(processor_data, real_estate_data,
+                                      main_schema, land_use_plans, contaminated_sites):
+
     request = MockRequest()
     request.matchdict.update(request_matchdict)
     request.params.update(request_params)
     processor = create_processor()
     webservice = PlrWebservice(request)
     params = webservice.__validate_extract_params__()
-    real_estate = processor.real_estate_reader.read(params, egrid=u'TEST3')
+    real_estate = processor.real_estate_reader.read(params, egrid=u'TEST2')
     extract = processor.process(real_estate[0], params, 'http://test.ch')
     plrs = extract.real_estate.public_law_restrictions
-    assert len(plrs) == 4
```

Comment:
@mki-c2c I think the goal of this test was to test the sorting mechanism according to the law statuses, but in the current test data we only have `InKraft` at the moment so this test is useless. I can prepare new test data

Reply:
Thanks, that's a good remark. So far, there is a lot of interdependent test data in the DB. Using local patches, we can customize the data for specific tests.